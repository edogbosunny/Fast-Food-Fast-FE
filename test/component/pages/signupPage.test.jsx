import React from 'react';
import { shallow } from 'enzyme';
import Signup from '../../../src/component/Pages/SignupPage';

describe('Login page', () => {
  const wrapper = shallow(
    <Signup />,
  );
  it('should mount the login page', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
