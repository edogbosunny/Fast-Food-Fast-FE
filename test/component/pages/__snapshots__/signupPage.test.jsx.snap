// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page should mount the login page 1`] = `<withRouter(Connect(SignupCard)) />`;
